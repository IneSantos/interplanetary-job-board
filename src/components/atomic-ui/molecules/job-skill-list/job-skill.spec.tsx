import React from "react";
import { render } from '@testing-library/react';
import SkillList from "./job-skill-list";

describe("SkillList atom tests", () => {
  test(`SkillList component render`, () => {
    const { container } = render(
      <SkillList skills={[]}/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
