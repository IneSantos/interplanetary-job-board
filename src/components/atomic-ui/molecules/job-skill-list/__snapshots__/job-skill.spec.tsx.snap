// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkillList atom tests SkillList component render 1`] = `
<div
  class="skill-list"
>
  <span
    class="skill"
  >
    This is a skill
  </span>
</div>
`;
